src/Client/Web/Bit.Client.Web.BlazorUI/Components/Links/BitLink.razor

```diff
@@ -1,24 +1,23 @@
 ﻿@namespace Bit.Client.Web.BlazorUI
 @inherits BitComponentBase
 
-@if (Href.HasValue() || !IsEnabled)
-{
-    <a @ref="RootElement"
-       style="@StyleBuilder.Value"
-       class="@ClassBuilder.Value"
-       href="@Href"
-       target="@Target">
-        @ChildContent
-    </a>
-}
-else
-{
-    <button @ref="RootElement"
-            style="@StyleBuilder.Value"
-            class="@ClassBuilder.Value"
-            type="button"
-            @onclick="HandleClick"
-            @onclick:preventDefault="true">
-        @ChildContent
-    </button>
-}
+<span @ref="RootElement"
```

Comment:
Is "Span" an appropriate root element in this situation?

Reply:
I chose span since in this case root element has to have an inline display and both< button> and <a> can be a child of span